src/wallet/wallet.h

```diff
@@ -581,7 +581,7 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
      */
     TransactionError FillPSBT(PartiallySignedTransaction& psbtx,
                   bool& complete,
-                  int sighash_type = 1 /* SIGHASH_ALL */,
+                  int sighash_type = 0 /* SIGHASH_DEFAULT */,
```

Comment:
Maybe I'm missing something obvious, but any reason why we use magic numbers with comments here (and also for the interface in `src/wallet/scriptpubkeyman.h`), rather than the enum directly?
```suggestion
                  int sighash_type = SIGHASH_DEFAULT,
```

Reply:
There was a time a while ago where that didn't work and this code has just been moved around to various places without changing. I've changed it to use `SIGHASH_DEFAULT` directly.